workspace-grid@mathematical.coffee.gmail.com/extension.js

```diff
@@ -308,7 +326,9 @@ function calculateWorkspace(direction, wraparound, wrapToSame) {
  *        https://extensions.gnome.org/extension/29/workspace-navigator/)
  */
 function moveWorkspace(direction) {
-    let newWs = actionMoveWorkspace(direction);
+    // if we should override the scroll direction
```

Comment:
Incomplete comment

Reply:
This is a boolean passed to the `actionMoveWorkspace` function. If **overrideScrollDirection** is true and **scroll-direction** is horizontal, it overrides the UP and DOWN directions to LEFT and RIGHT.

Here this behaviour is not needed because we are handling the keyboard arrow shortcuts and all directions are valid.

Reply:
Sorry I meant please complete the comment in the source code because `// if we should override the scroll direction` seems incomplete.